java/emitter.js

```diff
@@ -1,425 +1,479 @@
+                        type: 'StartInfo',
+                        schema: {
+                            batchInfo: {
+                                type: 'BatchInfo',
+                                schema: {
+                                    properties: {
+                                        type: 'List<Map<String, String>>',
+                                        schema: {
+                                            length: {rename: 'size'}
+                                        },
+                                        items: {
+                                            type: 'Map<String, String>',
+                                            items: {
+                                                type: 'String'
+                                            }
+                                        }
+                                    },
+                                }
+                            },
+                            agentRunId: 'String'
+                        }
+                    }
                 },
-                imageMatchSettings: imageMatchSettings,
-                knownVariantId: {
-                  type: 'String'
-                }
-              }},
-          },
-          startInfo: {
-          	type: 'StartInfo',
-          	schema: {
-          		batchInfo: {
-          			type: 'BatchInfo',
-          			schema: {
-      					properties: {
-        					type: 'List<Map<String, String>>',
-        					schema: {
-          						length: {rename: 'size'}
-          					},
-          					items: {
-          						type: 'Map<String, String>',
-          						items: {
-            						type: 'String'
-          						}
-        					  }
-        				  },
-      				  }
-      			  },
-              agentRunId: 'String'
-          	}
-          }
+            })
         },
-      })
-    },
-    getDom(result, domId) {
-      return addCommand(java`getDom(${result},${domId});`).type({type: 'JsonNode', recursive: true}).methods({
-        getNodesByAttribute: (dom, attr) => addCommand(java`getNodesByAttributes(${dom}, ${attr});`).type({
-          type: 'List<JsonNode>',
-          schema: {length: {rename: 'size'}},
-          items: {type: 'JsonNode', recursive: true}
-        })
-      })
+        getDom(result, domId) {
+            return addCommand(java`getDom(${result},${domId});`).type({type: 'JsonNode', recursive: true}).methods({
+                getNodesByAttribute: (dom, attr) => addCommand(java`getNodesByAttributes(${dom}, ${attr});`).type({
+                    type: 'List<JsonNode>',
+                    schema: {length: {rename: 'size'}},
+                    items: {
+                        type: 'JsonNode', schema: {
+                            rect: {
+                                type: 'rect',
+                                schema: {
+                                    top: 'Number',
+                                    left: 'Number'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
                                    left: "Number"
```